lib/js/atoms/Card/Card.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta, Story, Props } from '@storybook/addon-docs';
+import Card from './Card';
+
+<Meta title='atoms/Card' component={Card} />
+
+# Card
+
+<Story name='interactive'
+	   argTypes={{
+		   header: {control: { type: 'text' }, defaultValue: 'header slot'},
+		   content: {control: { type: 'text' }, defaultValue: 'content slot that supports <b>HTML markup</b>'},
+		   footer: {control: { type: 'text' }, defaultValue: 'footer slot'},
```

Comment:
They all support HTML markup, no?

Reply:
yes, but this one was important to show to UX